cabal.project

```diff
@@ -73,4 +75,13 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/Win32-network
-  tag: 3825d3abf75f83f406c1f7161883c438dac7277d
\ No newline at end of file
+  tag: 3825d3abf75f83f406c1f7161883c438dac7277d
+
+-- Needed for ghc-9.2.4
```

Comment:
Hmm. Should we push a patched version to CHaP? upstream looks fairly dead. Perhaps we can remove the dependency, but it is useful